src/lib/components/chat/ChatInput.svelte

```diff
@@ -21,6 +21,12 @@
 		// submit on enter
 		if (event.key === "Enter" && !event.shiftKey) {
 			event.preventDefault();
+			// blur to close keyboard on mobile
+			textareaElement.blur();
+			// refocus so that user on desktop can start typing without needing to reclick on textarea
+			if (innerWidth > TABLET_VIEWPORT_WIDTH) {
```

Comment:
What if we do a touch device check instead of screen width?  👀

Reply:
I think with `navigator.maxTouchPoints`  you'll have false positives on laptops with touch screens for example. But with the viewport solution you'll also have false positives on desktop browsers if they're sized super small. 

I think we would need the [VirtualKeyboard API](https://developer.mozilla.org/en-US/docs/Web/API/VirtualKeyboard_API) which is [not supported very well](https://developer.mozilla.org/en-US/docs/Web/API/VirtualKeyboard_API#browser_compatibility), or tracking changes in the [availHeight](https://developer.mozilla.org/en-US/docs/Web/API/Screen/availHeight) which feels overkill. So it's just which compromise we want to pick, I think viewport width is fine